scripts/west_commands/patch.py

```diff
@@ -344,7 +355,7 @@ def get_mod_paths(args, yml):
 
         mod_paths = {}
         for patch_info in patches:
-            mod = patch_info["module"]
+            mod = patch_info["module"].rstrip('/')
```

Comment:
Would it make sense to create a Path object out of mod instead of calling rstrip?